src/test/regress/sql/multi_utility_warnings.sql

```diff
@@ -6,3 +6,4 @@
 -- databases.
 
 CREATE DATABASE new_database;
+DROP DATABASE new_database;
```

Comment:
same here, this is needed to make this test idempotent because this PR needs to update its output for the sake of the changes introduced here